src/subtyping.md

```diff
@@ -1,146 +1,127 @@
 # Subtyping and Variance
 
-Subtyping is a relationship between types that allows statically typed
-languages to be a bit more flexible and permissive.
+Rust uses lifetimes to track the relationships between borrows and ownership.
+However, a naive implementation of lifetimes would be either too restrictive,
+or permit undefined behaviour.
 
-Subtyping in Rust is a bit different from subtyping in other languages. This
-makes it harder to give simple examples, which is a problem since subtyping,
-and especially variance, is already hard to understand properly. As in,
-even compiler writers mess it up all the time.
+In order to allow flexible usage of lifetimes
+while also preventing their misuse, Rust uses a combination of **Subtyping** and **Variance**.
 
-To keep things simple, this section will consider a small extension to the
-Rust language that adds a new and simpler subtyping relationship. After
-establishing concepts and issues under this simpler system,
-we will then relate it back to how subtyping actually occurs in Rust.
+## Subtyping
 
-So here's our simple extension, *Objective Rust*, featuring three new types:
+Subtyping is the idea that one type can be a *subtype* of another.
```

Comment:
```suggestion
Subtyping is the idea that one type can be used in place of another.
```

I'm not happy with the current line, because a reader probably doesn't know what a subtype is either.